### PR TITLE
Clarifying LaunchDarkly migration guide

### PR DESCRIPTION
**Pull request by bertilhatt**
Explicitly listing the variants that the developper will have to consider when transitioning.

### CODE REVIEW

File: docs/guides/migrations/launchdarily-migration.md [hunk 1]
```diff
@@ -136,7 +136,9 @@
           return assignment;
         }
         ```
-        
+        If you wnat to use JSON or Numeric variants, you will have
```
Comment (felipecsl) [inline docs/guides/migrations/launchdarily-migration.md:139]: typo `wnat`

Comment (felipecsl) [reply docs/guides/migrations/launchdarily-migration.md:139]: cc @bertilhatt

Comment (bertilhatt) [reply docs/guides/migrations/launchdarily-migration.md:139]: Thank you!

Comment (bertilhatt) [reply docs/guides/migrations/launchdarily-migration.md:139]: Fixed in https://github.com/Eppo-exp/eppo-docs/pull/461

**Comment by bertilhatt** [comment]
Continued https://github.com/Eppo-exp/eppo-docs/pull/461

**Pull request merged**